docs/resources/resource.md

```diff
@@ -55,7 +64,9 @@ resource "twingate_resource" "resource" {
 
 ### Optional
 
-- `group_ids` (Set of String) List of Group IDs that have permission to access the Resource, cannot be generated by Terraform and must be retrieved from the Twingate Admin Console or API
+- `access` (Block List, Max: 1) Restrict access to certain groups or service accounts (see [below for nested schema](#nestedblock--access))
+- `authoritative` (Boolean) Determines authoritative behaviour for handling the Resource's groups or service accounts.
```

Comment:
--> "Determines whether assignments in the `access` block will override any existing assignments. Default is `true`. If set to `false`, assignments made outside of Terraform will be ignored."